api/openapi-spec/swagger.json

```diff
@@ -1648,6 +1648,10 @@
     "io.k8s.api.apps.v1.StatefulSetSpec": {
       "description": "A StatefulSetSpec is the specification of a StatefulSet.",
       "properties": {
+        "persistentVolumeClaimDeletePolicy": {
+          "description": "PersistentVolumeClaimDeletePolicy indicates the PersistentVolumeClaimDeletePolicy that will determine when volumes from the VolumeClaimTemplates will be deleted when the controllering StatefulSet is deleted or scaled down.",
```

Comment:
Same comments from types.go. Guessing that this will get fixed once the swagger is generated again.

Reply:
Eventually I figured out swagger comes from staging/src/k8s.io/api/apps/v1/ and not pkg/apis/apps. Sigh.